frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -5,28 +5,28 @@ import { DashboardCardPopupLink } from '../dashboard-card/DashboardCardLink';
 import { HealthState, healthStateMapping, healthStateMessage } from './states';
 import { SecondaryStatus } from '../../status';
 
-const HealthItemIcon: React.FC<HealthItemIconProps> = ({ state }) => (
-  <div className="co-dashboard-icon">
+const HealthItemIcon: React.FC<HealthItemIconProps> = ({ state, dataTest }) => (
+  <div data-test={dataTest} className="co-dashboard-icon">
     {(healthStateMapping[state] || healthStateMapping[HealthState.UNKNOWN]).icon}
   </div>
 );
 
 const HealthItem: React.FC<HealthItemProps> = React.memo(
-  ({ className, state, title, details, popupTitle, noIcon = false, children }) => {
+  ({ className, state, title, details, popupTitle, noIcon = false, dataTest, children }) => {
     const { t } = useTranslation();
 
     const detailMessage = details || healthStateMessage(state, t);
 
     return (
-      <div className={classNames('co-status-card__health-item', className)}>
+      <div className={classNames('co-status-card__health-item', className)} data-test={dataTest}>
```

Comment:
cant we just generate the data-test value to be something like
```suggestion
      <div className={classNames('co-status-card__health-item', className)} data-test={`${title}-health-item`}>
```

we wouldnt need to pass the `dataTest` prop

Reply:
Fixed!